src/containers/ProjectInfo.js

```diff
@@ -6,29 +6,18 @@ import { Container } from 'semantic-ui-react'
 import ProjectSummary from '../components/ProjectSummary'
 
 export class ProjectInfo extends Component {
-  state = { project: null };
-
   componentDidMount () {
     const projectSlug = this.props.match.params.slug
     this.props.setLastLocation(this.props.location.pathname)
-    if (this.props.project.slug === this.props.match.params.slug) {
-      this.setState({ project: this.props.project })
-    } else {
+    if (this.props.project.slug !== this.props.match.params.slug) {
```

Comment:
this is much better logic @jmpainter... great job!

I did notice a small interval where clicking on a different project shows the previous project's info until the store is updated with the current project's info from the `fetchProjectInfo` action, but I used throttling to test it with regular 2g and it was noticeable, but probably not an issue.